frontend/src/pages/index.astro

```diff
@@ -1,10 +1,12 @@
 ---
 import HomeTitleBar from "../components/HomeTitleBar.astro";
+import CookieBanner from "../components/CookieBanner";
 import IndexLayout from "../layouts/IndexLayout.astro";
 ---
 
 <IndexLayout title="TheSchedule">
   <HomeTitleBar/>
+  <CookieBanner client:load></CookieBanner>
```

Comment:
This should be inside some kind of "root layout" which other Layouts use, since every site has to have this banner